leptonai/photon/photon.py

```diff
@@ -776,6 +776,12 @@ def favicon():
                 return Response(status_code=404)
             return FileResponse(path)
 
+        @app.get("/queue-length", include_in_schema=False)
+        def queue_length():
+            num_tasks = len(lepton_uvicorn_server.server_state.tasks)
+            # subtract 1 for the current task
+            return num_tasks or num_tasks - 1
```

Comment:
Actually, my stupid fault, it should be

return num_tasks and num_tasks - 1

...